web/src/i18n/locales/en.yaml

```diff
@@ -67,12 +67,12 @@
             _no: "No"
     overview: 
       title: Program Overview
-      helpText:
-      reminder: >-
-        Provide a brief, high level summary of the state program including history,
+      helpText: >-
+       Provide a brief, high level summary of the state program including history,
         and future vision. Include history and summary of future
-        vision of the state program. Indicate which federal fiscal years this
-        APD covers.
+        vision of the state program.
```

Comment:
The previous sentence sounds like exactly the same guidance.  If I'm reading this correctly in GH:
"Provide a brief, high level summary of the state program including history, and future vision. Include history and summary of vision of the state program."

The second sentence seems redundant.